src/rtc/h5_prep.py

```diff
@@ -303,7 +303,7 @@ def save_orbit(orbit, orbit_group, orbit_file_path):
     else:
         orbit_type = 'Undefined'
 
-    d = orbit_group.require_dataset("orbitType", (), "S10",
+    d = orbit_group.require_dataset("orbitType", (), "S25",
```

Comment:
Found that the string like `'POE precise orbit'` has cut out to `'POE precis'`. Same thing will happen for `'RES restituted orbit'` So increased the string length accordingly.

Reply:
Good catch! Thanks